### PR TITLE
Tax administrators

### PR DESCRIPTION
**Pull request by MarekBodingerBA**
chrome_zHLyJtl8L4

Data with unfortunately weird fix for ŤA:
```
[
    {
        "range": "A,BARCH",
        "name": "Bronislava Vojkovičová",
        "email": "bronislava.vojkovicova@bratislava.sk",
        "phone": "+421 2 59 356 948",
        "officeNumber": 1
    },
    {
        "range": "BARI,BIT",
        "name": "Zuzana Halušťoková",
        "email": "zuzana.halustokova@bratislava.sk",
        "phone": "+421 2 59 356 949",
        "officeNumber": 2
    },
    {
        "range": "BIŤ,BUJN",
        "name": "Ľubica Pašková",
        "email": "lubica.paskova@bratislava.sk",
        "phone": "+421 2 59 356 923",
        "officeNumber": 8
    },
    {
        "range": "BUJO,ČL",
        "name": "Mária Szedlárová",
        "email": "maria.szedlarova@bratislava.sk",
        "phone": "+421 2 59 356 951",
        "officeNumber": 4
    },
    {
        "range": "ČM,DULM",
        "name": "Martina Burzová",
        "email": "martina.burzova@bratislava.sk",
        "phone": "+421 2 59 356 952",
        "officeNumber": 5
    },
    {
        "range": "DULN,E,FLOR",
        "name": "Eva Hurbanová",
        "email": "eva.hurbanova@bratislava.sk",
        "phone": "+421 2 59 356 953",
        "officeNumber": 6
    },
    {
        "range": "FLOY,GODT",
        "name": "Jana Jakabová",
        "email": "jana.jakabova@bratislava.sk",
        "phone": "+421 2 59 356 922",
        "officeNumber": 7
    },
    {
        "range": "GODU,HART",
        "name": "Ľubica Pašková",
        "email": "lubica.paskova@bratislava.sk",
        "phone": "+421 2 59 356 923",
        "officeNumber": 8
    },
    {
        "range": "HARU,HORÚ",
        "name": "Ing. Janka Kövágó",
        "email": "janka.kovago@bratislava.sk",
        "phone": "+421 2 59 356 924",
        "officeNumber": 9
    },
    {
        "range": "HORV,CHRS",
        "name": "Monika Bugalová",
        "email": "monika.bugalova@bratislava.sk",
        "phone": "+421 2 59 356 925",
        "officeNumber": 10
    },
    {
        "range": "CHRT,JI",
        "name": "Eva Cudráková",
        "email": "eva.cudrakova@bratislava.sk",
        "phone": "+421 2 59 356 926",
        "officeNumber": 11
    },
    {
        "range": "JO,KHT",
        "name": "Zuzana Ďurčenková",
        "email": "zuzana.durcenkova@bratislava.sk",
        "phone": "+421 2 59 356 927",
        "officeNumber": 12
    },
    {
        "range": "KHU,KOPJ",
        "name": "Renáta Kovárová",
        "email": "renata.kovarova@bratislava.sk",
        "phone": "+421 2 59 356 928",
        "officeNumber": 13
    },
    {
        "range": "KOPK,KRÁL",
        "name": "Helena Bisová",
        "email": "helena.bisova@bratislava.sk",
        "phone": "+421 2 59 356 929",
        "officeNumber": 14
    },
    {
        "range": "KRAM,KUSD",
        "name": "Jana Bučeková",
        "email": "jana.bucekova@bratislava.sk",
        "phone": "+421 2 59 356 931",
        "officeNumber": 15
    },
    {
        "range": "KUSE,LOT",
        "name": "Jana Hrycková",
        "email": "jana.hryckova@bratislava.sk",
        "phone": "+421 2 59 356 932",
        "officeNumber": 16
    },
    {
        "range": "LOU,MARN",
        "name": "Ing. Renata Michálková",
        "email": "renata.michalkova@bratislava.sk",
        "phone": "+421 2 59 356 933",
        "officeNumber": 17
    },
    {
        "range": "MARO,MIJ",
        "name": "Ľubica Letavayová",
        "email": "lubica.letavayova@bratislava.sk",
        "phone": "+421 2 59 356 934",
        "officeNumber": 18
    },
    {
        "range": "MIK,MUSJ",
        "name": "Mgr. Mária Trebatická",
        "email": "maria.trebaticka@bratislava.sk",
        "phone": "+421 2 59 356 935",
        "officeNumber": 19
    },
    {
        "range": "MUSK,N,ON",
        "name": "Andrea Nitrianska",
        "email": "andrea.nitrianska@bratislava.sk",
        "phone": "+421 2 59 356 936",
        "officeNumber": 20
    },
    {
        "range": "OP,PERM",
        "name": "Marta Zelenková",
        "email": "marta.zelenkova@bratislava.sk",
        "phone": "+421 2 59 356 937",
        "officeNumber": 21
    },
    {
        "range": "PERN,PORU",
        "name": "Ľubica Brindzíková",
        "email": "lubica.brindzikova@bratislava.sk",
        "phone": "+421 2 59 356 938",
        "officeNumber": 22
    },
    {
        "range": "PORV,RIH",
        "name": "Jana Glebocká",
        "email": "jana.glebocka@bratislava.sk",
        "phone": "+421 2 59 356 940",
        "officeNumber": 23
    },
    {
        "range": "RICH,SCHT",
        "name": "Jana Ševecová",
        "email": "jana.sevecova@bratislava.sk",
        "phone": "+421 2 59 356 941",
        "officeNumber": 24
    },
    {
        "range": "SCHU,STCH",
        "name": "Bc. Veronika Oremusová",
        "email": "veronika.oremusova@bratislava.sk",
        "phone": "+421 2 59 356 942",
        "officeNumber": 25
    },
    {
        "range": "STI,ŠERD",
        "name": "Viera Kocourková",
        "email": "viera.kocourkova@bratislava.sk",
        "phone": "+421 2 59 356 939",
        "officeNumber": 26
    },
    {
        "range": "ŠERE,ŠUBCH",
        "name": "Katarína Sedláková",
        "email": "katarina.sedlakova@bratislava.sk",
        "phone": "+421 2 59 356 944",
        "officeNumber": 27
    },
    {
        "range": "ŠUBÍ,TREL",
        "name": "Gabriela Žáčiková",
        "email": "gabriela.zacikova@bratislava.sk",
        "phone": "+421 904 550 530",
        "officeNumber": 28
    },
    {
        "range": "TREM,TZ",
        "name": "Gabriela Hitmarová",
        "email": "gabriela.hitmarova@bratislava.sk",
        "phone": "+421 2 59 356 946",
        "officeNumber": 29
    },
    {
        "range": "ŤA,ŤA",
        "name": "Gabriela Žáčiková",
        "email": "gabriela.zacikova@bratislava.sk",
        "phone": "+421 904 550 530",
        "officeNumber": 28
    },
    {
        "range": "ŤB,VARG",
        "name": "Gabriela Hitmarová",
        "email": "gabriela.hitmarova@bratislava.sk",
        "phone": "+421 2 59 356 946",
        "officeNumber": 29
    },
    {
        "range": "VARH,WAGM",
        "name": "Jana Lepeyová",
        "email": "jana.lepeyova@bratislava.sk",
        "phone": "+421 2 59 356 920",
        "officeNumber": 30
    },
    {
        "range": "WAGN,X,Y,Z,Ž",
        "name": "Danka Šumcová",
        "email": "danka.sumcova@bratislava.sk",
        "phone": "+421 2 59 356 919",
        "officeNumber": 31
    }
]
```

### CODE REVIEW

File: strapi/src/components/tax-administrators/tax-administrator.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      "required": true
+    },
+    "phone": {
+      "type": "string",
+      "required": true
+    },
+    "officeNumber": {
+      "type": "integer",
+      "required": true,
+      "min": 1
```
Comment (radoslavzeman) [inline strapi/src/components/tax-administrators/tax-administrator.json:30]: I would probably go with string and regex here.

File: strapi/src/api/tax-administrators-list/content-types/tax-administrators-list/schema.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+  "kind": "singleType",
+  "collectionName": "tax_administrators",
+  "info": {
+    "singularName": "tax-administrators-list",
+    "pluralName": "tax-administrators",
```
Comment (radoslavzeman) [inline strapi/src/api/tax-administrators-list/content-types/tax-administrators-list/schema.json:6]: I would use name `taxAdministratorsList` in GUI and leave Strapi to generate others.

Comment (MarekBodingerBA) [reply strapi/src/api/tax-administrators-list/content-types/tax-administrators-list/schema.json]: Fixed.

File: strapi/src/components/tax-administrators/tax-administrator.json (hunk 1 above)
Comment (MarekBodingerBA) [reply strapi/src/components/tax-administrators/tax-administrator.json]: Fixed.

**Review by radoslavzeman — APPROVED** [review]

**Pull request merged**